integration-tests/src/test/java/oracle/kubernetes/operator/BaseTest.java

```diff
@@ -57,6 +57,7 @@
   private static String resultDir = "";
   private static String userProjectsDir = "";
   private static String projectRoot = "";
+  protected static String sitconfigDir = "";
```

Comment:
copyright statement at top of file is still wrong

Reply:
Fixed.